query/lambda/lambda_function_dead_letter_queue_configured.sql

```diff
@@ -13,4 +13,4 @@ select
   region,
   account_id
 from
-  aws_lambda_function;
\ No newline at end of file
+  "aws_lambda_function";
```

Comment:
@megaproaktiv Is there a reason you're adding quotes around the table name?  We don't typically do this, as our table names are usually Postgres friendly (with a few exceptions for dynamic tables).

Reply:
In several versions of steampipe, this caused errors due to postgres syntax